DOCS/man/vo.rst

```diff
@@ -262,8 +262,8 @@ Available video output drivers are:
     the same set of features as ``--vo=gpu``. See `GPU renderer options`_ for a
     list.
 
-    Currently, this only supports ``--gpu-api=vulkan``, and no hardware
-    decoding. Unlike ``--vo=gpu``, the FBO formats are not tunable, but you can
+    Currently, this does not support any hardware decoding.
+    Unlike ``--vo=gpu``, the FBO formats are not tunable, but you can
```

Comment:
Probably want to mention that it also doesn't currently support `--gpu-api=d3d11`.